Makefile

```diff
@@ -32,6 +33,7 @@ quality: ## Run Quality checks
 
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q pip-tools
+	pip-compile --rebuild --upgrade --allow-unsafe -o requirements/pip.txt requirements/pip.in
 	pip-compile --rebuild --upgrade -o requirements/dev.txt requirements/base.in requirements/dev.in
 	pip-compile --rebuild --upgrade -o requirements/test.txt requirements/base.in requirements/test.in
 	pip-compile --rebuild --upgrade -o requirements/travis.txt requirements/travis.in
```

Comment:
We should only need `--rebuild` on the first pip-compile run in the target.

Reply:
updated the pip-compile commands.